src/proposals/payout_proposal.cpp

```diff
@@ -124,14 +124,14 @@ namespace hypha
     {
         asset nonDeferredUsd = adjustAsset(usd, (float)1 - ((float)deferred / (float)100));
         // convert symbol from USD to HUSD
-        return asset{nonDeferredUsd.amount, common::S_HUSD};
+        return asset{nonDeferredUsd.amount, common::S_PEG};
     }
 
     asset PayoutProposal::calculateHypha(const asset &usd, const int64_t &deferred)
```

Comment:
There are some calculation functions here that are Hypha specific. I think this is OK, but we should also have generic versions of them.  At its simplest, the proposals should just have the token quantities explicit in the proposal itself.  The starting point should not have any type of interpretation of a USD input to the individual tokens.

Reply:
- [ ] This is a task, does it work?

Reply:
We could try moving from 'calculate the token based on a yearly usd salary' to 'set the exact amount of tokens received by period'.  Idk, we should have a discussion on this next stand-up or tech meeting